packages/core/types/common.d.ts

```diff
@@ -129,5 +129,5 @@ interface Request {
 export interface User {
   id?: string;
   email?: string;
-  name?: string
+  name?: string;
```

Comment:
We probably need to work out whether we're going with semi-colons or not for ts files and enforce it with a rule. Currently, as in this case, there is no linting rule, in contrast with the JS files.